src/routers/genericRouters/routerTransitiveClosure.ts

```diff
@@ -38,7 +32,7 @@ export class RouterTransitiveClosure implements Router {
     this.nodeCollectioName = nodeCollectionName
   }
 
-  async getPaths (from: string, to: string): Promise<any> {
+  async getPaths (from: string, to: string, fields: string[]): Promise<any> {
```

Comment:
Can you also add comment for this function? I am not quite sure what does this function do. Thank you!